model/registry/geo.yaml

```diff
@@ -0,0 +1,113 @@
+groups:
+  - id: registry.geo
+    type: attribute_group
+    prefix: geo
+    brief: >
+      Geo fields can carry data about a specific location related to an event.
+      This geolocation information can be derived from techniques such as Geo IP, or be user-supplied.
+    attributes:
+      - id: city_name
+        stability: experimental
+        type: string
+        brief: >
+          City name.
+        examples: [ 'Montreal', 'Berlin' ]
+      - id: continent_code
```

Comment:
Given the recent problems with attribute names with `_`, I think we should think about changing `continent` to be a namespace, specially since there's already multiple attributes under it. `continent.code|name`.